babel.config.js

```diff
@@ -11,11 +11,25 @@ function resolvePath(sourcePath, currentFile, opts) {
 
 let defaultPresets;
 
-// We release a ES version of Material-UI.
-// It's something that matches the latest official supported features of JavaScript.
-// Nothing more (stage-1, etc), nothing less (require, etc).
+// evergreen build
+// see https://next.material-ui.com/guides/minimizing-bundle-size/#evergreen-build
 if (process.env.BABEL_ENV === 'es') {
-  defaultPresets = [];
```

Comment:
It used to work with the previous stage approach of Babel.

Reply:
What worked?

Reply:
With the former stage preset strategy of Babel, we were only transpiling non official features. It was an approximation. The situation is better now with the env preset.